scripts.js

```diff
@@ -50,6 +52,99 @@ menuContactUs.addEventListener('click', () => {
+    popuptec: ['CSS', 'Html', 'Bootstrap', 'Ruby'],
+    popupimg: './pictures/golf.jpg',
+    btnlive: 'https://www.github.com',
+    btnsource: 'https://www.github.com',
+    btnName: 'btn1',
+    btnimg1: './pictures/TimeAndIcons/github.png',
+    btnimg2: './pictures/TimeAndIcons/github.png',
+  },
+  {
+    id: 1,
+    popuptitle: 'Multi-Post Stories',
+    popupdescription: 'This is the second A daily for this is a test and is test number 2',
+    popuptec: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+    popupimg: './pictures/beach.jpg',
+    btnlive: 'https://www.github.com',
+    btnsource: 'https://www.github.com',
+    btnName: 'btn3',
+    btnimg1: './pictures/TimeAndIcons/github.png',
+    btnimg2: './pictures/TimeAndIcons/github.png',
+  },
+  {
+    id: 2,
+    popuptitle: 'Multi-Post Stories',
+    popupdescription: 'This is the third A daily for this is a test and is test number 3',
+    popuptec: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+    popupimg: './pictures/desert.jpg',
+    btnlive: 'https://www.github.com',
+    btnsource: 'https://www.github.com',
+    btnName: 'btn4',
+    btnimg1: './pictures/TimeAndIcons/github.png',
+    btnimg2: './pictures/TimeAndIcons/github.png',
+  },
+];
+
+document.getElementById('card-works-container').innerHTML = arr1.map((items) => `<div class="card-works">  
+              <h2 class="card-works-title" id="card-works-title">${items.popuptitle}</h2>
+              <p class="card-works-description" id="card-works-description">${items.popupdescription}</p>
+              <ul class="card-works-tech">
+                <li class="card-works-tech-items">${items.popuptec[0]}</li>
+                <li class="card-works-tech-items">${items.popuptec[1]}</li>
+                <li class="card-works-tech-items">${items.popuptec[2]}</li>
+                <li class="card-works-tech-items">${items.popuptec[3]}</li>
+              </ul>
+              <button class="card-works-button project-2" type="button">
+                <a href="#popup">See Project</a>
+              </button>
+            </div>`).join('');
+
+// ---------------------------- OPEN POPUP WINDOW SECTION ----------------------------------
+
+const proj1 = {
+  title: 'Multi Post Stories',
+  img: './images/popup-img.png',
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry´s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+  techList: ['Html', 'Bootstrap', 'Ruby on rails'],
+};
+
+const tech1 = proj1.techList[0];
+const tech2 = proj1.techList[1];
+const tech3 = proj1.techList[2];
+
+const btn1 = document.querySelector('.project-1');
+const popupOpen = document.querySelector('.popup');
+
+btn1.addEventListener('click', () => {
```

Comment:
- [ ] As the previous reviewer mentioned, the popup only opens when you click on the `see project` button of **the first card. That means the remaining three project cards don't show the popup**. It may be because of that you have not added the click event which fires a function to open the popup on their `see project` button 💡. Please add that functionality so that the user will be able to see the details of every project you have in the works section of your portfolio 😃 😎.

Reply:
I can't deploy this feature, I hope to do that on weekend after PR approval!!! 😄 I'll have more time and less pressure.

Reply:
Popup working on all card-works 💪